README.md

```diff
@@ -44,15 +44,15 @@ The task-worker's role is to start and monitor the scraper's container for the t
 
 #### uploader
 
-The [uploader](https://ghcr.io/openzim/uploader) is instantiated by the task-worker to upload, individually, each created ZIM files, as well as the scraper's container log.
+The [uploader](https://ghcr.io/openzim/zimfarm-uploader) is instantiated by the task-worker to upload, individually, each created ZIM files, as well as the scraper's container log.
 
 The uploader supports both SCP and SFTP. We are currently using SFTP for all uploads due to a slight speed gain.
 
 Uploader is very fast and convenient (can watch and resumes files) but works only off files at the moment.
 
```

Comment:
Please add missing `dnscache`, explaining it is just a dnsmasq server ensuring specific nameservers are used and caching DNS results so that should DNS become instable it will not harm running tasks.